data-visualization/json/index01.js

```diff
@@ -0,0 +1,15 @@
+
+
+/*
+JavaScript Object Notation (JSON) is a text format that you use to structure data.
+
+Because JSON is lightweight, human readable, and easily parsed by machines, it's a popular structure to exchange data over the web.
+
+It started as a JavaScript standard, but now it is language independent and can be used in many programming languages.
+
+JSON is built using two structures:
+1. Objects: unordered collections of key-value pairs, surrounded by `{ }`
+2. Arrays: ordered collections of values, surrounded by `[ ]`
+
+Declare a new variable using the `let` keyword and name it `myTeam`, then set it equal to an empty object.
```

Comment:
```suggestion
Declare a new variable using the `const` keyword and name it `myTeam`, then set it equal to an empty object.
```
Would rather use const if we aren't reassigning something to this variable